### PR TITLE
Adding ruby solution and Readme.md

### PR DESCRIPTION
**Pull request by yvalenta**
- [X] Adding ruby solution and Readme.md

:rabbit:

### CODE REVIEW

File: yvalenta/chess_board.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Chess_board
+
+  def board(f,c)
+    #f = 7
```
Comment (rderoldan1) [inline yvalenta/chess_board.rb]: Remove unused code!

File: yvalenta/chess_board.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    #c = 8
+    col = 0
+    perro = []
```
Comment (rderoldan1) [inline yvalenta/chess_board.rb]: Please, use useful names for constants and variables!

File: yvalenta/chess_board.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        end
+      end
+      perro[col] = ñ
+      col = col+1
+    end
+    perro
+  end
+end
+
+## Derechos de autor de las siguiente 3 lineas (@rderoldan1)
```
Comment (rderoldan1) [inline yvalenta/chess_board.rb]: :+1:

**Comment by yvalenta** [comment]
is only the before
and the unused code is to see the differences when the arguments are send by the user and when the arguments are burned
#Sorry4MyEnglish

:ear:

File: yvalenta/chess_board.rb (hunk 1 above)
Comment (rderoldan1) [inline yvalenta/chess_board.rb]: If you like you hardcode the values is better to use it in the params definition

Comment (yvalenta) [inline yvalenta/chess_board.rb]: tururú.. :S

**Pull request merged**